jsonschema-core.xml

```diff
@@ -800,49 +800,257 @@
+                        changing the meta-schema partway through the validation for non-schema
+                        instances.
+                    -->
+                </t>
+                <t>
+                    Values for this property are defined in other documents and by other parties.
+                    JSON Schema implementations SHOULD implement support for current and previous
+                    published drafts of JSON Schema vocabularies as deemed reasonable.
+                </t>
+            </section>
+            <section title='The "$vocabulary" Keyword'>
+                <t>
+                    The "$vocabulary" keyword is used within meta-schemas to declare
+                    the sets of keywords that schemas described by the meta-schema are
+                    expected to use.
+                </t>
+                <t>
+                    The value of this keyword MUST be an object.  The property names in the
+                    object MUST be URIs, (containing a scheme) and this URI MUST be normalized.
+                    The URI MAY be a URL, but the nature of the retrievable resources is
+                    currently undefined, and reserved for future use.  Vocabulary authors
+                    SHOULD NOT serve a document at that URL.  A server MAY respond with
+                    the relevant protocol's successful "no content" message, such as
+                    an HTTP 204 status.
+                    <cref>
+                        Vocabulary documents may be added shortly, or in the next draft.
+                        For now, identifying the keyword set is deemed sufficient as that,
+                        along with meta-schema validation, is how the current "vocabularies"
+                        work today.
+                    </cref>
+                </t>
+                <t>
+                    The values of the object properties MUST be booleans.
+                    If the value is true, then implementations that do not recognize
+                    the vocabulary MUST refuse to process any schemas that declare
+                    this meta-schema with "$schema".  If the value is false, implementations
+                    that do not recognize the vocabulary MAY choose to proceed with processing
+                    such schemas.
+                </t>
+                <t>
+                    When processing a schema that uses unrecognized vocabularies, keywords
+                    declared by those vocabularies are treated like any other unrecognized
+                    keyword, and ignored.
+                </t>
+                <t>
+                    The "$vocabulary" keyword SHOULD be used in the root schema of any schema
+                    document intended for use as a meta-schema.  It MUST NOT appear in subschemas.
+                </t>
+                <t>
+                    The "$vocabulary" keyword MUST be ignored in a schema documents that
```

Comment:
documents / document

Reply:
Yeah, although I think I'd rather remove the "a" and go with "in schema documents"- does that make sense?  I'm never sure what's better with this sort of thing.